packages/gatsby/src/schema/graphql-engine/standalone-regenerate.ts

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+/*
+this is used for development purposes only
+to be able to run `gatsby build` once to source data
+and print schema and then just rebundle graphql-engine
+with source file changes and test re-built engine quickly
+
+Usage:
+There need to be at least one successful `gatsby build`
+before starting to use this script (warm up datastore,
+generate "page-ssr" bundle). Once that's done you can
+run following command in test site directory:
+
+```shell
+node node_modules/gatsby/dist/schema/graphql-engine/standalone-regenerate.js
+```
+*/
```

Comment:
this module/script is not part of actual fix (it's not actually used by gatsby itself anywhere), but it proved really handy (again and again), so I think it's time to just commit this as this might be useful not just for myself

Reply:
Agreed, thank you!